selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/NavigationBar.java

```diff
@@ -81,12 +81,12 @@ public void clickOnMenu(MenuItem menuItem) {
     seleniumWebDriver.findElement(By.xpath(locator)).click();
   }
 
-  public String getMenuCounterValue(MenuItem menuItem) {
+  public int getMenuCounterValue(MenuItem menuItem) {
```

Comment:
Please, add example of menu item and returning value into JavaDoc to explain the propose of method.